WPFHospitalEditor/Equipment.xaml.cs

```diff
@@ -0,0 +1,151 @@
+﻿using HealthcareBase.Dto;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Media;
+
+namespace WPFHospitalEditor
+{
+    /// <summary>
+    /// Interaction logic for Equipment.xaml
+    /// </summary>
+    public partial class Equipment : Window
+    {
+        Dictionary<String,EquipmentDto> allEquipment = new Dictionary<string, EquipmentDto>();
```

Comment:
Please change name to EquipmentWindow or something like that.

Reply:
fixed